app-crypt/mc-agent/mc-agent-0.0.1.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+EGO_PN=github.com/raoulh/${PN}
+
+DESCRIPTION="Simple ssh-agent that loads keys stored from Moolticute"
+HOMEPAGE="https://github.com/raoulh/mc-agent"
+EGIT_COMMIT="29cf340ec129f5b70bc5d6f8abb9a53406d07305"
+SRC_URI="https://github.com/raoulh/mc-agent/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz
+	https://github.com/jawher/mow.cli/archive/2f22195f169da29d54624afd9eb83ada5c9e4ee9.tar.gz -> mow.cli.tar.gz
+	https://github.com/gorilla/websocket/archive/21ab95fa12b9bdd8fecf5fa3586aad941cc98785.tar.gz -> websocket.tar.gz
+	https://github.com/satori/go.uuid/archive/36e9d2ebbde5e3f13ab2e25625fd453271d6522e.tar.gz -> go.uuid.tar.gz
+	https://github.com/golang/crypto/archive/2d027ae1dddd4694d54f7a8b6cbe78dca8720226.tar.gz -> crypto.tar.gz
+"
+
+KEYWORDS=""
+LICENSE="GPL-3"
+SLOT="0"
+
+DEPEND="dev-lang/go:0"
```

Comment:
why? there's eclassed for that

Reply:
Hey! sorry this is my first ebuild file. When you say that there's an eclassed for that I assume you mean by using inherit golang (or whatever the eclass is) it will be magically handled?